Podfile.lock

```diff
@@ -1,36 +1,20 @@
 PODS:
-  - OnTheRoad (0.9.0)
-  - Pelias (1.0.0-beta4):
-    - Pelias/Core (= 1.0.0-beta4)
-  - Pelias/Core (1.0.0-beta4)
+  - OnTheRoad (1.0.0)
+  - Pelias (1.0.0):
+    - Pelias/Core (= 1.0.0)
+  - Pelias/Core (1.0.0)
```

Comment:
Why is there a separate line for `Pelias/Core` in the lock file but not other dependencies and not in any other config files?